Hazelcast.Net/Hazelcast.Config/NearCacheConfig.cs

```diff
@@ -77,101 +83,198 @@ public virtual NearCacheConfigReadOnly GetAsReadOnly()
             return _readOnly;
         }
 
+        /// <summary>
+        /// Returns the eviction policy for the Near Cache.
+        /// </summary>
+        /// <returns>eviction policy</returns>
         public virtual string GetEvictionPolicy()
         {
             return _evictionPolicy;
         }
 
+        /// <summary>
+        /// Returns <see cref="InMemoryFormat"/> 
+        /// </summary>
+        /// <returns><see cref="InMemoryFormat"/> </returns>
         public virtual InMemoryFormat GetInMemoryFormat()
         {
             return _inMemoryFormat;
         }
 
+        /// <summary>
+        /// Returns the maximum number of seconds each entry can stay in the Near Cache as untouched (not-read).
+        /// Entries that are not read (touched) more than <c>maxIdleSeconds</c> value will get removed from the Near Cache.
```

Comment:
(I think it should be like: /// Entries that are not read (untouched) more than...

(Or just like this: /// Entries that are not read more than...